storage/kv/badger.go

```diff
@@ -249,11 +233,13 @@ func (b *BadgerDB) listRawKeys(prefix []byte, txn *badger.Txn) [][]byte {
 
 	opt := badger.DefaultIteratorOptions
 	opt.Prefix = prefix
+	opt.PrefetchValues = false
```

Comment:
As far as I understand `allGetter` implementation, yes. it seems that this is used only to get the keys and then `Get` is called on them again to get the values. While the option mentioned above is used for the iterator. ideally if we would use the iterator to get all the values as well it will be more efficient to remove this option. but that is out of the scope of this PR IMO.